cwf/gateway/docker/docker-compose.override.yml

```diff
@@ -33,7 +33,7 @@ services:
 
   sessiond:
     environment:
-      MAGMA_PRINT_GRPC_PAYLOAD: 1
```

Comment:
revert?

Reply:
oh i remove it on purpose. Are you really using it? Normally i only enable it when debugging locally one test at the time, otherwise is impossible to differentiate each test at the logs.

But if you find it useful to have it enable on ci, i can revert it

Reply:
oh this is the prod one, yeah let's turn it off for that. I would rather keep it enabled for integ tests though. It is useful for debugigng CI failures.